ui/main/src/app/api/user.api.ts

```
@@ -0,0 +1,36 @@
+                },
+                getAllEntities: function () {
+                    const entities = [];
+                    EntitiesService.getEntities().forEach((entity) => entities.push({...entity}));
+                    return entities;
+                }
+            }
+        };
+        // prevent unwanted modifications from templates code
+        Object.freeze(opfab.users);
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)